site/app/models/User.php

```diff
@@ -451,6 +455,10 @@ public function setRegistrationSection($section) {
         $this->registration_section = ($section !== null) ? $section : null;
     }
 
+    public function CourseSectionId($course_section_id) {
+        $this->course_section_id = ($course_section_id !== null) ? $course_section_id : null;
+    }
+
```

Comment:
This should probably be called `setCourseSectionId`. The null check also is not necessary, the same thing happens without it. Please ask one of the people who is more familiar with PHP about the setter and getter functions not needing to be explicitly defined, I remember seeing them come from some comment at the top of the file.

Reply:
Yeah, I renamed it and added it as a "magic" function, and it works the same.